styleguide/derek/pattern-components/challenges-river/_challenges-river.scss

```diff
@@ -22,11 +23,42 @@
   }
 }
 
+.challenges-learnMore {
+  color: $rackspace-red;
+  display: inline-block;
+  font-size: 1em;
+  font-weight: 400;
+}
+
 .challenges-item {
   @include make-xs-column(10);
   @include make-md-column(8);
+  height: 100%;
+  padding-bottom: 10px;
+  padding-top: 10px;
+  transition-duration: .2s;
+
+  .challenges-itemRackShape {
+    transition-duration: .2s;
+  }
```

Comment:
Ehhh sorry, one last thing here. We should be able to move this `transition-duration` to the non-nested version of this selector (line 71).  I understand the need to keep it nested for the actual `transform` below due to it responding to the parent's hover state, but as far as the duration itself, it should be fine to live at the top-level.